lib/src/model.dart

```diff
@@ -5140,15 +5148,16 @@ class PackageBuilder {
     if (uri != null) {
       source = new FileBasedSource(javaFile, uri);
```

Comment:
Do we still use java files?

Reply:
Apparently so.  It looks like it might be possible to eliminate some of the usage (the TODO above), but we still need them for finding the analysis errors we care about later on.